### PR TITLE
 Added C8555 Test case to MT

### PR DESCRIPTION
**Pull request by wMercadoJala**

### CODE REVIEW

File: IDT.MoneyTransfer/Main/UI/Components/TabMoneyTransferComponent.cs [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+        }
+
+        /// <summary>
+        /// Gets a new instance from SentderForm to the press the optionMoneyTransfer button.
+        /// </summary>
+        /// <returns>SenderForm</returns>
+        public PayerLocationsLookupPage ClickTabPayerLocationsLookup()
+        {
+            CommonActions.ClickElement(this.TabPayerLocationsLookup);
+            string test = Main.Resources.Locators.CashPickUpFormLocators.BtnSave;
```
Comment (DavidVallejosT) [inline IDT.MoneyTransfer/Main/UI/Components/TabMoneyTransferComponent.cs]: Is this variable being used? If not, please remove it.

File: IDT.MoneyTransfer/Main/UI/Components/TabMoneyTransferComponent.cs [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+        [IDTFindBy(File = "TabMoneyTransferLocators", Locator = "TabQuickRateLookup")]
+        private IWebElement TabQuickRateLookup;
+
+        [IDTFindBy(File = "TabMoneyTransferLocators", Locator = "TabPayerLocationsLookup")]
+        private IWebElement TabPayerLocationsLookup;
+
+        [IDTFindBy(File = "TabMoneyTransferLocators", Locator = "TabCustomersManagement")]
+        private IWebElement TabCustomersManagement;
+
+        public TabMoneyTransferComponent()
```
Comment (DavidVallejosT) [inline IDT.MoneyTransfer/Main/UI/Components/TabMoneyTransferComponent.cs]: Add Document to this constructor.

File: IDT.MoneyTransfer/Main/UI/Pages/MoneyTransfer/PayerLocationsLookup/PayerLocationsLookupPage.cs [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+    using System.Collections.Generic;
+    using IDT.Core.Automation.Page.Base;
+    using IDT.Core.Automation.Selenium;
+    using IDT.Core.Automation.Utilities;
+    using IDT.Core.Automation.Utilities.CommonAction;
+    using IDT.Core.Automation.WebDriver;
+    using OpenQA.Selenium;
+
+    /// <summary>
+    /// test
```
Comment (DavidVallejosT) [inline IDT.MoneyTransfer/Main/UI/Pages/MoneyTransfer/PayerLocationsLookup/PayerLocationsLookupPage.cs]: Please improve this description.

File: IDT.MoneyTransfer/Main/UI/Components/TabMoneyTransferComponent.cs (hunk 2 above)
Comment (wMercadoJala) [reply IDT.MoneyTransfer/Main/UI/Components/TabMoneyTransferComponent.cs]: done

File: IDT.MoneyTransfer/Main/UI/Components/TabMoneyTransferComponent.cs (hunk 1 above)
Comment (wMercadoJala) [reply IDT.MoneyTransfer/Main/UI/Components/TabMoneyTransferComponent.cs]: done

File: IDT.MoneyTransfer/Main/UI/Pages/MoneyTransfer/PayerLocationsLookup/PayerLocationsLookupPage.cs (hunk 3 above)
Comment (wMercadoJala) [reply IDT.MoneyTransfer/Main/UI/Pages/MoneyTransfer/PayerLocationsLookup/PayerLocationsLookupPage.cs]: done

**Pull request merged**